demisto_sdk/commands/download/downloader.py

```diff
@@ -236,6 +237,63 @@ def download_playbook_yaml(self, playbook_string) -> str:
 
         return playbook_string
 
+    def handle_incidentfield(self, incidentfield_string: str, scripts_mapper: dict) -> str:
+        # In case the incident field uses a custom script, replace the id value of the script with its name
+        file_json_object = json.loads(incidentfield_string)
+        incidentfield_name = file_json_object.get('name')
+        if incidentfield_name and (incidentfield_name in self.input_files or self.all_custom_content):
+            if file_json_object.get('script') and file_json_object.get('script') in scripts_mapper:
+                incidentfield_string = incidentfield_string.replace(file_json_object.get('script'),
+                                                                    scripts_mapper[file_json_object.get('script')])
+
+        return incidentfield_string
+
+    def handle_layout(self, layout_string, scripts_mapper):
+        # In case the layout uses a custom script, replace the id value of the script with its name
+        file_json_object = json.loads(layout_string)
+        layout_name = file_json_object.get('name')
+        if layout_name and (layout_name in self.input_files or self.all_custom_content):
+            tabs = file_json_object.get('detailsV2', {}).get('tabs', [])
+            for tab in tabs:
+                sections = tab.get('sections', [])
+                for section in sections:
+                    items = section.get('items', [])
+                    for item in items:
+                        if item.get('scriptId') and item.get('scriptId') in scripts_mapper:
+                            layout_string = layout_string.replace(item.get('scriptId'),
+                                                                  scripts_mapper[item.get('scriptId')])
+
+        return layout_string
+
+    @staticmethod
+    def map_script(script_string: str, scripts_mapper: dict) -> dict:
+        script_yml = yaml.load(script_string)
+        script_id = script_yml.get('commonfields').get('id')
+        if re.search(r'^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$',
```

Comment:
Please extract the regex expression to the `constants.py` file, ideally reusing the other existing `constants` to construct the `regex`.